templates/lzv-rights/discrimination-lzv.jade

```diff
@@ -9,44 +9,59 @@ html
 		include ../includes/header.jade
 		
 				
-		#recorddiv68552395.t-cover(bgimgfield='img', style="height:50vh; background-image:url('../img/lzv-rights/HEAD/1.jpg');")
-			#coverCarry68552395.t-cover__carrier.loaded(data-content-cover-id='68552395', data-content-cover-height='50vh', data-content-cover-parallax='', style='height: 50vh; background-attachment: scroll; background-image: url("../img/lzv-rights/HEAD/1.jpg");', src='')
-			.t-cover__filter(style='height:50vh;background-color:#000;filter:alpha(opacity:40);KHTMLOpacity:0.4;MozOpacity:0.4;opacity:0.4;')
+		#recorddiv68552395.t-cover(bgimgfield='img', style="height:300px; background-image:url('../img/lzv-rights/HEAD/1.jpg');")
```

Comment:
I think it isn't the best practice to name `id`'s and classes like that. Should we also avoid using inline styles? (no inline styles, CSS police will come for you)
The whole usage below is the same.